relnotes/default-integration.migration.md

```diff
@@ -1,2 +1,3 @@
 * Integration tests default location changed from `test` to `integrationtest`.
-  The location can be set explicitly via the variable `INTEGRATIONTEST`.
+
+  The location can be set explicitly via the variable `INTEGRATIONTEST`, but you are not required to define it anymore (as it was required in v1.x.x), you just need to define it if you don't want the new default (`integrationtest`).
```

Comment:
Sometimes I'm wondering if release notes should address the user directly, or use a more passive tense, e.g. "it is not required to define it...".

Reply:
The never ending dilemma when writing documentation in general...